heudiconv/convert.py

```diff
@@ -176,7 +177,7 @@ def prep_conversion(sid, dicoms, outdir, heuristic, converter, anon_sid,
         assure_no_file_exists(edit_file)
         write_config(edit_file, info)
         save_json(filegroup_file, filegroup)
-
+    import pdb;pdb.set_trace()
```

Comment:
we don't need this one do we? ;)

Reply:
ha. nope. definitely a work in progress :)